pkg/services/anonymous/anonimpl/impl.go

```diff
@@ -109,11 +193,20 @@ func (a *AnonDeviceService) TagDevice(ctx context.Context, httpReq *http.Request
 		LastSeen:  time.Now().UTC(),
 	}
 
+	err = a.tagDeviceUI(ctx, httpReq, *taggedDevice)
```

Comment:
10.2 should not be released with 2 tagging methods, the tag/untag mechanisms generates too many calls already on a fresh boot